python/src/ops.cpp

```diff
@@ -3638,62 +3638,96 @@ void init_ops(py::module_& m) {
         )pbdoc");
   m.def(
       "atleast_1d",
-      &atleast_1d,
-      "a"_a,
+      [](const py::args& args,
+         StreamOrDevice s) -> std::variant<py::object, py::list> {
```

Comment:
Do you need to declare the `variant` output type here? I might just have it all be `py::object`?

Reply:
returning only py::object is sufficient, thanks for that. I thought a variant was necessary when returning a python list.